flashcards/backend/decks/tests/test_models_tag.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from ..models import Tag
+
+
```

Comment:
This is tested in the test_tag_save_method_in_db test. I checked by turning off the custom save method and adding a solution with a Lower. Test works fine :)

Reply:
So I have a question because I think tags do not need to be lowercased, for example, I would prefer to have a tag `JavaScript` instead of just `javascript`, but it is just and idea 😅

Reply:
The assumption was that the tags could not be repeated with different capitalization. But of course there is one more option - we can hardcode the list of tags and that's it :) . However, I think that only the basic functionality needs to be set in the first version.

Reply:
ok, so when we are talking about the minimal implementation, then having unique lower constraint will prevent typlication for `Python`, and `python`. Meaning, if you save `Python` tag you won't be able to save `python`, and if you choose to save `python` then you won't be able to save `Python`. This is what the unique lower constraint is doing. That said I think saving only lowercased tags is not needed, but I might be wrong 😅

Reply:
Ha, amazing, it's work :) . I just had to delete the custom save. Thanks :smile: